doc/scheduling_devgd.md

```diff
@@ -0,0 +1,80 @@
+# Scheduling in TempestaFW: Developer Guide
```

Comment:
I didn't review the file since it doesn't affect end user experience. If you'd imagine that the doc would be useful for you when you join the project, then I'm OK with it. Feel free to add any useful information for new comers (experienced core developers however). Just please place the developer docs separately from user guide.